src/AppBundle/Manager/ReportManager.php

```diff
@@ -116,17 +119,33 @@ private function decideTo(Report $report, $status, $comment, $decidedAt = null)
         $decision
             ->setDecidedAt($decidedAt ?: new \DateTime())
             ->setStatus($status)
-            ->setComment($comment)
-        ;
+            ->setComment($comment);
 
         return $report;
     }
 
+    public function getClassifications($locale = null)
+    {
+        $locale = $locale ?: $this->requestStack->getCurrentRequest()->getLocale();
+        $reflection = new \ReflectionClass(Report::class);
+        $filter = function ($value) {
+            return false !== strpos($value, 'CLASSIFICATION_');
+        };
+        $constants = array_filter($reflection->getConstants(), $filter, ARRAY_FILTER_USE_KEY);
```

Comment:
factorize this part of code with ``AppBundle\Utils\Utils::getConstants($startsWith)`` static method